test/fixtures/eslint-warnings/webpack.config.js

```diff
@@ -11,19 +11,18 @@ module.exports = {
 
   ],
   module: {
-    preLoaders: [
+    loaders: [
```

Comment:
I'm using the new 'preLoaders' syntax, i.e. changing `preLoaders` to `enforce: 'pre'` -- I think Webpack 2 is going to support the `loaders` syntax at least until `2.1.0`.